http-agent/src/main/resources/db/migration/V1_MOTECH-2502.sql

```diff
@@ -0,0 +1,3 @@
+-- change type of field "responseBody" to text
```

Comment:
@mgrabowski7 Actually, this entity has been just recently added, so we can skip writing a specific migration to fix that column type.